modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/cluster/services/Cluster.scala

```diff
@@ -55,6 +57,7 @@ object Cluster {
           }
           clusterId = clusterStorage.getClusterId
           state <- nodeStorage.getNodeState
+          _ = if (state === Ready) MonadThrow[F].raiseError[ClusterSessionToken](InvalidNodeState)
```

Comment:
there should be no check like that in this place. anyway, the syntax here is also wrong. `_ <- ...`